src/courtroom.cpp

```diff
@@ -2939,7 +2968,11 @@ void Courtroom::on_ooc_return_pressed()
       return;
 
   }
-
+  else if(ooc_message.startsWith("/clearooc"))
+  {
+      ui_server_chatlog->clear();
```

Comment:
This is not enough. See the data structure that is used for exporting logs.

Reply:
After exporting the logs it does the same command. Not sure what you mean.